docs/deploy/aws.md

```diff
@@ -90,10 +84,13 @@ Please refer to [AWS documentation](https://aws.amazon.com/dynamodb/getting-star
    **Note:** It's preferable to run the binary as a service using systemd or your operating system's facilities.
 
 ### On ECS
+
 To support container-based environments like AWS ECS, lakeFS can be configured using environment variables. Here are a couple of `docker run` 
 commands to demonstrate starting lakeFS using Docker:
+
 #### With PostgreSQL
-```sh
+
+```bash
```

Comment:
1. What's the difference? (from markdown POV)
2. Why was it changed here and left as `sh` in the next section?

Reply:
1. bash is equal to sh syntax as far as I know - can revert
2. just added blank lines